server_admin/topics/threat/admin.adoc

```diff
@@ -93,4 +93,29 @@ Equivalent configuration using CLI commands:
 
 /subsystem=undertow/configuration=filter/expression-filter=portAccess:add(,expression="path-prefix('/auth/admin') and not equals(%p, 8444) -> response-code(403)")
 /subsystem=undertow/server=default-server/host=default-host/filter-ref=portAccess:add()
-----
\ No newline at end of file
+----
+
+==== Server version
+
+There are scenarios where people may want to hide the {project_name} version. It is possible to create Undertow filters redirecting all the requests to `/auth/version` to any page you want, preventing people to know the details about the server. The following example do not permit access to `/auth/version`:
+
```

Comment:
Edit suggestion:
There are scenarios where it is desirable to hide the {project_name} version. To do so, create Undertow filters redirecting all the requests to `/auth/version` to any page you want, which will then prevent access to the details about the server. The following example prevents access to `/auth/version`: